docs/Contributing/design-qa-considerations.md

```diff
@@ -0,0 +1,113 @@
+# Design / QA considerations
+
+This is meant to be a helpful checklist of 'events' or 'transactions' to remember to think about while designing or testing new features or bugs to make sure edge cases are caught sooner rather than later. Please feel free to add more if any are missing
+
+## User
+
+- Create User
```

Comment:
```suggestion
- Create user
```

Looking good! I think let's update this to use sentence case everywhere: https://fleetdm.com/handbook/company/why-this-way#why-does-fleet-use-sentence-case